### PR TITLE
IAC - Adding Home Office API config.

### PR DESCRIPTION
**Pull request by raghera**
Adding new app config for AAT only and updating CODEOWNERS for it.

### CODE REVIEW

File: k8s/aat/common/ia/home-office-integration-api.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-home-office-integration-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
```
Comment (timja) [inline k8s/aat/common/ia/home-office-integration-api.yaml:8]: The new annotations should be used for this, search fluxcd in the repo

File: k8s/aat/common/ia/home-office-integration-api.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    name: ia-home-office-integration-api
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ia/home-office-integration-api:prod-503867d9
+      ingressHost: ia-home-office-integration-api-aat.service.core-compute-aat.internal
+      environment:
+        RESTART_ME: "0"
```
Comment (timja) [inline k8s/aat/common/ia/home-office-integration-api.yaml:26]: Not needed

File: k8s/aat/common/ia/home-office-integration-api.yaml (hunk 1 above)
Comment (raghera) [reply k8s/aat/common/ia/home-office-integration-api.yaml]: I've changed ... not sure if I needed others I saw on `draft-store` like: ```    repository.fluxcd.io/smokecron: java.smoketestscron.image
    filter.fluxcd.io/smokecron: glob:prod-*
    repository.fluxcd.io/functionalcron: java.functionaltestscron.image
    filter.fluxcd.io/functionalcron: glob:prod-*
    repository.fluxcd.io/smoke: java.smoketests.image
    filter.fluxcd.io/smoke: glob:prod-*
    repository.fluxcd.io/functional: java.functionaltests.image
    filter.fluxcd.io/functional: glob:prod-*```

File: k8s/aat/common/ia/home-office-integration-api.yaml (hunk 2 above)
Comment (raghera) [reply k8s/aat/common/ia/home-office-integration-api.yaml]: removed.  Is there another way to trigger restart of pods instead now?

Comment (timja) [reply k8s/aat/common/ia/home-office-integration-api.yaml]: the same, just add when required

**Pull request merged**